### DISCUSSION TITLE
Aurora Store

### DISCUSSION THREAD

**Discussion by Nichtraucher** [ImranR98/Obtainium#1389] [General]
Hello there :-)

The update broke for me because the Aurora dev moved the apk source from gitlab to https://auroraoss.com/AuroraStore/Stable/
Installing it from that source returns a "forbidden". :-/ I tried all kinds of options but had no luck. 

Any help is much appreciated! :-D

**Comment by DwainZwerg** [comment]
Hello @Nichtraucher ,
it's a known issue (https://github.com/ImranR98/Obtainium/issues/1384, https://github.com/ImranR98/Obtainium/issues/1382, https://github.com/ImranR98/Obtainium/issues/1381, https://github.com/ImranR98/Obtainium/issues/1380, https://github.com/ImranR98/Obtainium/issues/1359, https://github.com/ImranR98/Obtainium/issues/854, https://github.com/ImranR98/Obtainium/issues/785, https://github.com/ImranR98/Obtainium/issues/697). Retrieving updates from https://auroraoss.com/AuroraStore/Stable/ has not worked for a very long time (≈ 1/2 to 1 year) because cloudflare prohibits it. Now the AuroraStore devs no longer publish the [GitLab releases](https://gitlab.com/AuroraOSS/AuroraStore/-/releases) on GitLab, but link to [their own site](https://auroraoss.com/AuroraStore/Stable/), which, as already explained above, Obtainium cannot access due to Cloudflare's blocking. A possible workaround would be to track new versions via F-Droid (can be entered as a source in Obtainium).

**Reply by DwainZwerg** [reply]
You can of course also ask the devs if they would publish on GitLab again in the future, or if they can set their cloudflare protection differently.

**Reply by Nichtraucher** [reply]
Oops, sorry! I didn't search in the issues because I figured that it's a new one and unrelated to Obtainium.

**Reply by DwainZwerg** [reply]
Hello @Nichtraucher  ,
According to https://gitlab.com/AuroraOSS/AuroraStore/-/issues/1052#note_1778388859, the Aurora Store devs have fixed the problem. The download of AuroraStore should work normally again.

**Comment by Flashwalker** [comment]
Obtanium unable to find a release on https://auroraoss.com/downloads/AuroraStore/Release
